pages/common/license.md

```diff
@@ -0,0 +1,19 @@
+# license
+
+> Create license files for open-source projects.
+
+- Create a license:
```

Comment:
@rubenvereecken that's right. but probably it makes sense to mention this [algorithm](https://github.com/nishanths/license#customize-name-and-year-on-the-license) somehow, if it is possible to do it shortly.